pusher/src/Services/SocketManager.ts

```diff
@@ -199,7 +188,7 @@ export class SocketManager implements ZoneEventListener {
                 joinRoomMessage.addCharacterlayer(characterLayerMessage);
             }
 
-            console.log("Calling joinRoom '" + client.roomId + "'");
+            console.log("Calling joinRoom");
```

Comment:
You are removing a commit I made to add more details here. Are you sure you did not force push something not intended?